### PR TITLE
Ingest WOSP

### PR DESCRIPTION
**Pull request by mjpost**

### CODE REVIEW

File: data/xml/2020.wosp.xml [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+      <address>Wuhan, China</address>
+      <month>05 August</month>
+      <year>2020</year>
+      <url hash="6a1860b8">2020.wosp-1</url>
+    </meta>
+    <frontmatter>
+      <url hash="337783c9">2020.wosp-1.0</url>
+    </frontmatter>
+    <paper id="1">
+      <title>1</title>
```
Comment (danielgildea) [inline data/xml/2020.wosp.xml:21]: All the titles are missing?

File: data/xml/2020.wosp.xml [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+      <author><first>Shubhanshu</first><last>Mishra</last></author>
+      <author><first>Sudhanshu</first><last>Mishra</last></author>
+      <pages>65–70</pages>
+      <abstract>We present our team Scubed’s approach in the 3C Citation Context Classification Task, Subtask B, citation context influence classification. Our approach relies on text based features transformed via tf-idf features followed by training a variety of simple models resulting in a strong baseline. Our best model on the leaderboard is a random forest classifier using only the citation context text. A replication of our analysis finds logistic regression and gradient boosted tree classifier to be the best performing model. Our submission code can be found at: https://github.com/napsternxg/Citation_Context_Classification.</abstract>
+      <url hash="466bb438">2020.wosp-1.10</url>
+    </paper>
+    <paper id="11">
+      <title>2</title>
+      <author><first>Premjith</first><last>B</last></author>
+      <author><first>Soman</first><last>Kp</last></author>
```
Comment (danielgildea) [inline data/xml/2020.wosp.xml:109]: I think this is first initials and last names.

File: data/xml/2020.wosp.xml (hunk 1 above)
Comment (mjpost) [reply data/xml/2020.wosp.xml:21]: I pinged the chairs.

Comment (mjpost) [reply data/xml/2020.wosp.xml]: Fixed now I think.

File: data/xml/2020.wosp.xml (hunk 2 above)
Comment (mjpost) [reply data/xml/2020.wosp.xml:110]: Here's what the paper has:

image

Comment (mjpost) [reply data/xml/2020.wosp.xml]: He does have [an Anthology page](https://www.aclweb.org/anthology/people/s/soman-kp/)

**Review by danielgildea — APPROVED** [review]

**Pull request merged**